From 9b8ac6ae46c2dc974d0aee592d1549d70438bd44 Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Thu, 20 Oct 2022 13:05:53 -0700
Subject: [PATCH 2/3] Find whatever version of dragonbox

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 6313193b6..a16627e2d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -10533,7 +10533,7 @@ if test "$with_system_dragonbox" = "yes"; then
     AC_LANG_PUSH([C++])
     save_CPPFLAGS=$CPPFLAGS
     # This is where upstream installs to, unfortunately no .pc or so...
-    DRAGONBOX_CFLAGS=-I/usr/include/dragonbox-1.1.3
+    DRAGONBOX_CFLAGS=-I`readlink -f /usr/include/dragonbox-* | head -1`
     CPPFLAGS="$CPPFLAGS $DRAGONBOX_CFLAGS"
     AC_CHECK_HEADER([dragonbox/dragonbox.h], [],
        [AC_MSG_ERROR([dragonbox/dragonbox.h not found. install dragonbox])], [])
-- 
2.43.0

