From e6c5e232d868a3d9c9d5b8790cc28a4c970d103a Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Fri, 6 Nov 2020 09:12:37 -0800
Subject: [PATCH 2/2] Remove leftover cast of fResolutionDPI

---
 vcl/source/filter/ipdf/pdfread.cxx | 1 -
 1 file changed, 1 deletion(-)

diff --git a/vcl/source/filter/ipdf/pdfread.cxx b/vcl/source/filter/ipdf/pdfread.cxx
index 3066c4deba4a..93c88d64b799 100644
--- a/vcl/source/filter/ipdf/pdfread.cxx
+++ b/vcl/source/filter/ipdf/pdfread.cxx
@@ -218,7 +218,6 @@ size_t RenderPDFBitmaps(const void* pBuffer, int nSize, std::vector<Bitmap>& rBi
     (void)rBitmaps;
     (void)nFirstPage;
     (void)nPages;
-    (void)fResolutionDPI;
     return 0;
 #endif // HAVE_FEATURE_PDFIUM
 }
-- 
2.29.2

